Sources/TuistCore/Graph/Graph.swift

```diff
@@ -464,9 +464,9 @@ extension Graph {
 
     // swiftlint:disable:next line_length
     func frameworkUsesDynamicLinking(frameworkMetadataProvider: FrameworkMetadataProviding = FrameworkMetadataProvider()) -> (_ frameworkNode: PrecompiledNode) -> Bool { { frameworkNode in
-            let isDynamicLink = try? frameworkMetadataProvider.linking(precompiled: frameworkNode) == .dynamic
-            return isDynamicLink ?? false
-        }
+        let isDynamicLink = try? frameworkMetadataProvider.linking(precompiled: frameworkNode) == .dynamic
+        return isDynamicLink ?? false
```

Comment:
do you really want to swallow potential exceptions?

Reply:
That's a good point. Ideally we should throw an error here but we can do it as part of another PR.